src/mbgl/map/transform_state.cpp

```diff
@@ -151,14 +193,19 @@ CameraOptions TransformState::getCameraOptions(optional<EdgeInsets> padding) con
         .withPitch(pitch * util::RAD2DEG);
 }
 
+#pragma mark - EdgeInsets
+
+void TransformState::setEdgeInsets(const EdgeInsets& val) {
```

Comment:
Discussed with @zmiao offline: let's introduce a common setter accepting a newly introduced `TransformStateProperties` object, that might be a fluent interface.